### ISSUE TITLE
[FEATURE] Permitir ver a los administradores de un concurso cuánto tiempo le queda a cada concursante

### ISSUE THREAD

**Issue by lhchavez** [omegaup/omegaup#691]
@lhchavez a asignar apropiadamente para definir diseño de UI nueva

**Comment by sarvesh211999** [comment]
Is this issue still to be fixed ??

**Comment by lhchavez** [comment]
yes, if it's open, it still needs fixing. This is basically present a new (Vue) UI where an admin can see the times in which each participant of a contest they own started participating and displays how much time they have left in the contest (for contests that start a the participant's clock when they join).

This only applies to contests that have a non-`NULL` `window_length`: https://github.com/omegaup/omegaup/blob/bbff8d4550006bd6db3cb3c5a91b476bf9ae8dbc/frontend/database/schema.sql#L142
